### PR TITLE
Add several operators

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

**Comment by fzaninotto** [comment]
The documentation doesn't list all operators. Could you make it more explicit?

File: src/Collection.js [hunk 1]
```diff
@@ -54,11 +54,46 @@ const getSimpleFilter = (key, value) => {
         let realKey = key.replace(/(_gt)$/, '');
         return item => get(item, realKey) > value;
     }
+    if (key.indexOf("_neq_any") !== -1) {
+      // equal
```
Comment (fzaninotto) [inline src/Collection.js:58]: bad comment?

File: src/Collection.js [hunk 2]
```diff
@@ -54,11 +54,46 @@ const getSimpleFilter = (key, value) => {
...
+      let finalValue = Array.isArray(value) ? value : [value];
+      return (item) => finalValue.some((val) => get(item, realKey) == val);
+    }
+    if (key.indexOf('_eq') !== -1) {
+        // equal
+        let realKey = key.replace(/(_eq)$/, '');
+        return item => get(item, realKey) == value;
+    }    
+    if (key.indexOf('_inc_any') !== -1) {
+        // equal
```
Comment (fzaninotto) [inline src/Collection.js:80]: bad comment?

File: src/Collection.js [hunk 3]
```diff
@@ -54,11 +54,46 @@ const getSimpleFilter = (key, value) => {
...
+        let realKey = key.replace(/(_inc_any)$/, "");
+        let finalValue = Array.isArray(value) ? value : [value];
+        return item => finalValue.some(val => get(item, realKey).includes(val));
+    }
+    if (key.indexOf('_inc_all') !== -1) {
```
Comment (fzaninotto) [inline src/Collection.js:85]: I'd call it `inc` or `includes` for consistency with other operators

File: package-lock.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "fakerest",
-  "version": "3.3.0",
+  "version": "3.4.0",
```
Comment (fzaninotto) [inline package-lock.json:3]: You modified the wrong file...

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 4 above)
Comment (djhi) [reply package-lock.json]: npm did it

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**